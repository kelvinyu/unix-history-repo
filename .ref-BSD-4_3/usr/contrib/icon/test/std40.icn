record array(a,b,c,d,e,f,g)

procedure dummy(u,v,x,y,z)
   suspend u | v
   return x
end

procedure main()
   write("'a' ----> ",image('a') | "none")
   write("'ab' ----> ",image('ab') | "none")
   write("'\\xb9' ----> ",image('\xb9') | "none")
   write("'\\xb8\\xb4' ----> ",image('\xb8\xb4') | "none")
   write("'\\^d' ----> ",image('\^d') | "none")
   write("'\\^a\\^d' ----> ",image('\^a\^d') | "none")
   write("\"a\" ----> ",image("a") | "none")
   write("\"ab\" ----> ",image("ab") | "none")
   write("\"\\xb9\" ----> ",image("\xb9") | "none")
   write("\"\\xb8\\xb4\" ----> ",image("\xb8\xb4") | "none")
   write("\"\\^d\" ----> ",image("\^d") | "none")
   write("\"\\^a\\^d\" ----> ",image("\^a\^d") | "none")
   write("*'a' ----> ",image(*'a') | "none")
   write("*'ab' ----> ",image(*'ab') | "none")
   write("*'\\xb9' ----> ",image(*'\xb9') | "none")
   write("*'\\xb8\\xb4' ----> ",image(*'\xb8\xb4') | "none")
   write("*'\\^d' ----> ",image(*'\^d') | "none")
   write("*'\\^a\\^d' ----> ",image(*'\^a\^d') | "none")
   write("*\"a\" ----> ",image(*"a") | "none")
   write("*\"ab\" ----> ",image(*"ab") | "none")
   write("*\"\\xb9\" ----> ",image(*"\xb9") | "none")
   write("*\"\\xb8\\xb4\" ----> ",image(*"\xb8\xb4") | "none")
   write("*\"\\^d\" ----> ",image(*"\^d") | "none")
   write("*\"\\^a\\^d\" ----> ",image(*"\^a\^d") | "none")
   write("+\"a\" ----> ",image(+"a") | "none")
end
