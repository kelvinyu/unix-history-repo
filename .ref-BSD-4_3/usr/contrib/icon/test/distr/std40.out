'a' ----> 'a'
'ab' ----> 'ab'
'\xb9' ----> '\271'
'\xb8\xb4' ----> '\264\270'
'\^d' ----> '\004'
'\^a\^d' ----> '\001\004'
"a" ----> "a"
"ab" ----> "ab"
"\xb9" ----> "\271"
"\xb8\xb4" ----> "\270\264"
"\^d" ----> "\004"
"\^a\^d" ----> "\001\004"
*'a' ----> 1
*'ab' ----> 2
*'\xb9' ----> 1
*'\xb8\xb4' ----> 2
*'\^d' ----> 1
*'\^a\^d' ----> 2
*"a" ----> 1
*"ab" ----> 2
*"\xb9" ----> 1
*"\xb8\xb4" ----> 2
*"\^d" ----> 1
*"\^a\^d" ----> 2
Run-time error 102 at line 33 in std40.icn
numeric expected
offending value: "a"
